### ISSUE TITLE
Regarding feature alignment between EEG and images

### ISSUE THREAD

**Issue by wlc1256630** [ChiShengChen/MUSE_EEG#2]
Hello, I would like to ask for your advice. Do you have such a question: Is there any inherent connection between the image features extracted by the pre trained CLIP model and the EEG features extracted by the EEG encoder? I have read several articles on EEG Image matching, and the EEG data they use is the EEG signal changes caused by visual stimuli collected through the RSVP paradigm. What I am puzzled about is that our human eyes should pay more attention to the obvious information such as color and shape presented in the short time of image stimuli, and the image information extracted by CLIP may not necessarily be the color or other information we want. So how are these two established during feature alignment? Although similarity calculation is used for matching, I feel it is a kind of hard alignment. There is no inherent connection between these two modal data.

**Comment by ChiShengChen** [comment]
@wlc1256630 Thanks for your feedback. You're right that in multimodal learning, finding a natural connection between different types of data, like EEG and images, is a big challenge. The questioner pointed out that the link between image features and EEG features may not be that clear, and this is a common issue in current multimodal research. Plus, we can't yet treat the brainwaves generated by looking at an image as the actual "ground truth."

In trying to solve this, using CLIP assumes that EEG can somehow express similar semantics to the image labels in CLIP’s training set, which is used to align EEG, semantics, text, and images. But as you mentioned, this assumption might not be reliable. We probably need a deeper understanding of how the features of EEG and images are distributed, and more advanced strategies for cross-modal alignment. There’s also room to explore how psychology or neuroscience could help explain the connection between the two better.

That said, at least this approach allows for some early experiments, like EEG-to-image [1]. But in the future, for more advanced mind-to-anything applications, we’ll need better datasets and pretrained models with finer alignment to really make progress.

[1] https://arxiv.org/abs/2410.00712

**Comment by wlc1256630** [comment]
Thank you for your reply. This means that this is a consensus in the current research field. Will we study deeper internal connections in the future?

**Comment by ChiShengChen** [comment]
@wlc1256630 yes, we will dive deeper in the future
